readme.txt

```diff
@@ -4,9 +4,9 @@ Tags: woocommerce, export, delivery, packages, PostNL, Shipping
 Requires at least: 4.6
 Requires PHP: 5.6
 Tested up to: 6.3
-Stable tag: 5.3.1
+Stable tag: 5.3.0
```

Comment:
It seems this branch is not being merged with the latest `main` branch? Please try to merge `main` branch to this branch first.

Reply:
I already merged the main with this branch. The translations are also in this branch now